packages/logger/src/writers/logtape.ts

```diff
@@ -0,0 +1,36 @@
+import { getLogger, Logger as LogTapeLogger } from "@logtape/logtape";
+import { LogLevel } from "../logger";
+import { Attributes } from "../utils";
+import { LogWriter } from "./common";
+
+export interface LogTapeLogWriterOptions {
+	category?: Parameters<typeof getLogger>[0];
+	getProperties?(
+		level: LogLevel,
+		attrs: Attributes | null | undefined,
+		msg: string | null | undefined,
+	): Record<string, unknown>;
+}
+
+export class LogTapeLogWriter implements LogWriter {
+	#logTapeLogger: LogTapeLogger;
+
+	constructor(public options: LogTapeLogWriterOptions) {
+		this.#logTapeLogger = getLogger(this.options.category ?? ["graphql-hive"]);
+	}
+
+	write(
+		level: LogLevel,
+		attrs: Attributes | null | undefined,
+		msg: string | null | undefined,
+	): void {
+		const log = this.#logTapeLogger[level];
```

Comment:
The method should be bound:

```suggestion
		const log = this.#logTapeLogger[level].bind(this.#logTapeLogger);
```